test/index.js

```diff
@@ -351,132 +285,96 @@ test('excess leading dashes on options are retained', (t) => {
     positionals: []
   };
   const result = parseArgs({ strict: false, args: passedArgs, options: passedOptions });
-
-  t.deepEqual(result, expected, Error('excess option dashes are retained'));
-
-  t.end();
+  assert.deepStrictEqual(result, expected, Error('excess option dashes are retained'));
 });
 
 // Test bad inputs
 
-test('invalid argument passed for options', (t) => {
+test('invalid argument passed for options', () => {
   const passedArgs = ['--so=wat'];
   const passedOptions = 'bad value';
-
-  t.throws(() => { parseArgs({ args: passedArgs, options: passedOptions }); }, {
+  assert.throws(() => { parseArgs({ args: passedArgs, options: passedOptions }); }, {
     code: 'ERR_INVALID_ARG_TYPE'
   });
-
-  t.end();
 });
 
-test('then type property missing for option then throw', (t) => {
+test('then type property missing for option then throw', () => {
```

Comment:
```suggestion
test('type property missing for option then throw', () => {
```

Reply:
Oops, thanks. That was meant to be a when/then, so `test('when type property missing for option then throw', () => {`